packages/pass-style/src/make-far.js

```diff
@@ -179,7 +186,7 @@ export const Far = (farName, remotable = undefined) => {
     // * does further mutations,
     // * hardens the mutated object before returning it.
     // so this mutation is not unprecedented. But it is surprising!
-    r[GET_METHOD_NAMES] = getMethodNamesMethod;
+    Object.defineProperty(r, GET_METHOD_NAMES, getMethodNamesDescriptor);
```

Comment:
attn @erights

Reply:
Yes, non-enumerable is better. Resolving.